### PR TITLE
Logging docs

### PR DESCRIPTION
**Pull request by chemistry-sourabh**

### CODE REVIEW

File: docs/rest_api.md [hunk 1]
```diff
@@ -1,253 +1,289 @@
...
 
 BMI API assumes that you know the image which you are interested in. BMI has an driver which can communicate with Ceph(the file storage system that we are using for storing our images). Please make sure that you understand Ceph concepts before you play with the API. 
 
 To have a clear understanding of the API, we wish to provide the following terminology:
-
-project - A project is the HaaS project that has been allocated to tenant.
-node - A node that is allocated to project by HaaS. This node should be present in HaaS.
-img - The name of image which will be used for provisioning.
-snap_name - The snapshot of the image from which you want to provision a node.
+* project - A project is the HaaS project that has been allocated to tenant.
```
Comment (ravisantoshgudimetla) [inline docs/rest_api.md]: Change reference to HIL

File: docs/rest_api.md [hunk 2]
```diff
@@ -1,253 +1,289 @@
...
-- Internal 500 with some junk characters. This means the request body is not proper.
-- 401. This means unauthorized access to ceph image or snapshot or image already exists in ceph.
-- 403. This means a ceph connection problem.
-- 409. Image busy exception.
-- 444. You used a wrong request method like PUT instead of POST etc.
+ "project" : "<project_name>",
+ "node" : "<node_name>" , 
+ "img" : "<image_name>" ,
+ "network" : "<network_name>" ,
+ "channel" : "<channel in haas>" ,
```
Comment (ravisantoshgudimetla) [inline docs/rest_api.md]: HaaS to HIL again.

File: docs/rest_api.md [hunk 3]
```diff
@@ -1,253 +1,289 @@
...
 {
- "node" : "cisco-2016" ,
- "img" : "hadoopMaster.img" ,
- "project" : "bmi_infra" ,
- "snap_name" : "HadoopMasterGoldenImage"
+ "project" : "bmi_infra",
+ "node" : "cisco-2016" , 
+ "img" : "hadoopMaster.img" ,  
+ "network" : "provision-net" ,
+ "channel" : "vlan/native",
```
Comment (ravisantoshgudimetla) [inline docs/rest_api.md]: Do we need to specify channel? Shouldn't it be always native as long as we use HIL.

**Comment by ravisantoshgudimetla** [comment]
@chemistry-sourabh ,Done with my first pass, will do one more after discussing.

File: docs/rest_api.md [hunk 4]
```diff
@@ -1,253 +1,289 @@
...
--an HTTP method and URL path
--Request body(which will always be a JSON object)
--A list of possible responses
--An example
-
-Provision:
-
+---
+###Provision:
 Provision API is needed for provisioning a node from MOC cluster as of now. Provision operation internally calls a ceph clone operation. Ceph clone operation usually takes a snap_
```
Comment (ravisantoshgudimetla) [inline docs/rest_api.md]: I think there is a typo here? Takes a snap_?

**Comment by chemistry-sourabh** [comment]
Changed HaaS to HIL changes and Removed the channel (Removed it in the API call, but forgot to remove in docs)

**Comment by naved001** [comment]
I am closing this because we merged Sourabh's TGT branch to the dev branch #20

**Pull request closed without merging**